src/app/features/categories/categories.component.html

```diff
@@ -0,0 +1,12 @@
+<main>
+  <h2>Kategorie</h2>
+  <app-search [placeholderValue]="'Wyszukaj'"></app-search>
+  <app-categories-card
+    [categories$]="categories$"
+    [isAdmin$]="isUserAdmin$"
```

Comment:
tutaj zamiast sobie wydzielać poszczególne elementy stanu można było zasubkrybować się na cały stan (to komponent store także mamy pewność że i tak będziemy używać wszystkich składowych) i potem przekazać jego niezbędne cześci `state.categories` , `state.isUserAdmin` do komponentów prezentujących. Dzięki temu nasz komponent kontener ma konkretną odpowiedzialność, pobrania danych i przekazania ich niżej